config.json

```diff
@@ -4,5 +4,8 @@
     "httpsEnabled": true,
     "certFile": null,
     "keyFile": null,
-    "debug": false
+    "debug": false,
+    "callbackUrl": "http://172.31.128.1:9988/api/current/ucsCallback",
```

Comment:
port and IP is hardcoded

Reply:
I set this API as a default API, which is from our common configure file in:
https://github.com/RackHD/RackHD/blob/master/docker/monorail/config.json